platform/src/main/java/org/stellar/anchor/platform/custody/fireblocks/FireblocksPaymentService.java

```diff
@@ -46,11 +50,12 @@ public class FireblocksPaymentService implements CustodyPaymentService<Transacti
   private static final String QUERY_PARAM_SORT = "sort";
   private static final String TRANSACTIONS_ORDER_BY = "createdAt";
   private static final String TRANSACTIONS_SORT = "ASC";
-  public static int TRANSACTIONS_LIMIT = 500;
+  private static final int TRANSACTION_LIMIT = 500;
 
   private final FireblocksApiClient fireblocksApiClient;
   private final FireblocksConfig fireblocksConfig;
   private final Type transactionDetailsListType;
+  private int transactionLimit = TRANSACTION_LIMIT;
```

Comment:
I think we are doing this because we want to configure this for tests. If so, can we make `transactionLimit` a constructor parameter?

Reply:
Yes, this was added to allow configuration of the tests. 

Add a constructor that accepts the `transactionLimit` as an argument.